### PR TITLE
Removing/updating references to bootstrapped property

### PR DESCRIPTION
**Pull request by teallarson**

### CODE REVIEW

**Review by bleonard — APPROVED** [review]

**Review by bleonard — COMMENTED** [review]
I see some more references to "Boostrapped proeprty" in tableSOurce and others.
Try searching the codebase for "bootstrapp" (to not get the react ones.

There's some code in tableSource and in product concepts and in the tutorial.

**Comment by teallarson** [comment]
> I see some more references to "Boostrapped proeprty" in tableSOurce and others.
> Try searching the codebase for "bootstrapp" (to not get the react ones.
> 
> There's some code in tableSource and in product concepts and in the tutorial.

Thanks.  Got those now.  Did you check the accuracy as well?  A few places I added some clarification/rephrasing, but I want to be sure it's fully accurate.

File: components/docs/plugins/tableSource.tsx [hunk 1]
```diff
@@ -206,38 +189,21 @@ bootstrappedProperty: {
...
           Profiles
         </Link>
-        . In many cases, this is an ID type of value. Let's say you have a
-        column called <code>id</code> in your database, and you wanted to create
-        a Property called <code>user_id</code> in Grouparoo. And that Property (
-        <code>user_id</code>) would be the primary key for all Profiles. Then
-        your config might look something like this.
+        . In most cases, this is an ID or an e-mail address. Grouparoo will
+        automatically determine what field is used for a primary key depending
+        on the source and automatically generate one Profile per unique value in
```
Comment (bleonard) [inline components/docs/plugins/tableSource.tsx]: could say "based on the mapping" instead of "depending on the source"

**Review by bleonard — APPROVED** [review]
wording looks good to me! One suggestion

**Pull request merged**